app/containers/Login/actions.js

```diff
@@ -42,24 +42,26 @@ export function showEmailPasswordForm(val) {
   };
 }
 
-// Login with scatter
+// Login with Wallet
 
-export function loginWithScatter() {
+export function loginWithWallet({ keycat, scatter }) {
```

Comment:
optional: I see several places in the code where 2 boolean values are passed to indicate keycat vs scatter. Enums can simplify that logic and allow to store a single value. Also enum would not allow to send both values as true - keycat and scatter.